workloads/fio.yml

```diff
@@ -0,0 +1,130 @@
+          dest: "{{ansible_user_dir}}/scale-ci-tooling/workload-storage-fio-env.yml"
+
+    - name: Check if scale-ci-tooling namespace exists
+      shell: |
+        oc get projects | grep "scale-ci-tooling"
+      ignore_errors: true
+      changed_when: false
+      register: scale_ci_tooling_ns_exists
+    - name: print status of scale_ci_tooling_ns_exists
+      debug:
+        var: scale_ci_tooling_ns_exists
+
+    - name: Ensure any stale scale-ci-storage-fio job is deleted
+      shell: |
+        oc delete job scale-ci-storage-fio -n scale-ci-tooling
+      register: scale_ci_tooling_project
+      failed_when: scale_ci_tooling_project.rc == 0
+      until: scale_ci_tooling_project.rc == 1
+      retries: 60
+      delay: 1
+      when: scale_ci_tooling_ns_exists.rc == 0
+
+    - name: Block for non-existing tooling namespace
+      block:
+        - name: Create tooling namespace
+          shell: |
+            oc create -f {{ansible_user_dir}}/scale-ci-tooling/scale-ci-tooling-ns.yml
+      when: scale_ci_tooling_ns_exists.rc != 0
+
+    - name: Create tooling service account
+      shell: |
+        oc create serviceaccount useroot -n scale-ci-tooling
+        oc adm policy add-scc-to-user privileged -z useroot -n scale-ci-tooling
+      when: enable_pbench_agents|bool
+
+    - name: Add serviceaccount for useroot when pbench is not used
+      shell: |
+        oc delete serviceaccount -n scale-ci-tooling useroot 
+        oc create serviceaccount useroot -n scale-ci-tooling
+      when: not enable_pbench_agents|bool
+
+    - name: Create/replace kubeconfig secret
+      shell: |
+        oc replace --force -n scale-ci-tooling -f "{{ansible_user_dir}}/scale-ci-tooling/kubeconfig-secret.yml"
+
+    - name: Create/replace the pbench configmap
+      shell: |
+        oc replace --force -n scale-ci-tooling -f "{{ansible_user_dir}}/scale-ci-tooling/pbench-cm.yml"
+
+    - name: Create/replace pbench ssh secret
+      shell: |
+        oc replace --force -n scale-ci-tooling -f "{{ansible_user_dir}}/scale-ci-tooling/pbench-ssh-secret.yml"
+
+    - name: Create/replace workload script configmap
+      shell: |
+        oc replace --force -n scale-ci-tooling -f "{{ansible_user_dir}}/scale-ci-tooling/workload-fio-script-cm.yml"
+
+    - name: Create/replace workload script environment configmap
+      shell: | 
+        oc replace --force -n scale-ci-tooling -f "{{ansible_user_dir}}/scale-ci-tooling/workload-storage-fio-env.yml"
+
+    - name: Create/replace workload job to that runs workload script
+      shell: |
+        oc replace --force -n scale-ci-tooling -f "{{ansible_user_dir}}/scale-ci-tooling/workload-job.yml"
+
+    - name: Poll until job is running
+      shell: |
+        oc get pods --selector=job-name=scale-ci-storage-fio -n scale-ci-tooling -o json
+      register: pod_json
+      retries: 60
+      delay: 2
+      until: pod_json.stdout | from_json | json_query('items[0].status.phase==`Running`')
```

Comment:
I think there might have been some confusion regarding how the end of a workload playbook should be/look like.

Basically we need to wait until the job pod is running, then "poll until the job is complete".  Basically it should just resemble:

```
    - name: Poll until job pod is running
      shell: |
        oc get pods --selector=job-name=scale-ci-storage-fio -n scale-ci-tooling -o json
      register: pod_json
      retries: 60
      delay: 2
      until: pod_json.stdout | from_json | json_query('items[0].status.phase==`Running`')

    - name: Poll until job is complete
      shell: |
        oc get job scale-ci-storage-fio -n scale-ci-tooling -o json
      register: job_json
      retries: "{{job_completion_poll_attempts}}"
      delay: 10
      until: job_json.stdout | from_json | json_query('status.succeeded==`1` || status.failed==`1`')
      failed_when: job_json.stdout | from_json | json_query('status.succeeded==`1`') == false
      when: job_completion_poll_attempts|int > 0
```